modules/mesaides/templates/nginx_config.erb

```diff
@@ -118,12 +118,20 @@ server {
   }
 
   location @default {
-    proxy_pass        <%= @proxied_endpoint %>;
+    proxy_pass        http://<%= @upstream_name %>;
     proxy_redirect    off;
   }
   <% else %>
   location / {
-    proxy_pass        <%= @proxied_endpoint %>;
+
+    proxy_set_header X-Real-IP       $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host            $http_host;
+    proxy_set_header Connection      "";
```

Comment:
Can you add a comment to explicit the link with the `keepalive` directive in upstream ?